storage_v2/tracestore/reader.go

```diff
@@ -22,6 +22,7 @@ type Reader interface {
 	// Chunking requirements:
 	// - A single ptrace.Traces chunk MUST NOT contain spans from multiple traces.
 	// - Large traces MAY be split across multiple, *consecutive* ptrace.Traces chunks.
+	// - Each returned ptrace.Traces object must not be empty.
```

Comment:
use normative language
```suggestion
	// - Each returned ptrace.Traces object MUST NOT be empty.
```